komodo/build.py

```diff
@@ -327,7 +327,7 @@ def resolve(x):
             builddir=builddir,
             makeopts=current.get("makeopts"),
             makefile=current.get("makefile"),
-            dlprefix=dlprefix if dlprefix else ".",
+            dlprefix=dlprefix,
```

Comment:
Since this is called  `cache` in `cli.py,` should we rename it here as well?

Reply:
What happens if someone calls this `make() `now with dlprefix set to an empty string?

Reply:
(the function signature defaults it to `None`)

Reply:
Good point, changing `dlprefix` to `cache` for consistency. 

If the empty string is passed in, then `pushd` in `shell.py` fails when it tries to CD to '' in `fetch.py`. I think it should probably be a positional argument for `make`, and we throw an exception if it's Falsey.

Reply:
...and on further inspection, I'm not so sure. The problem is that the same parameter is called `outdir` in `komodo.fetch.fetch()`, and `output` in the CLI implemented by `komodo.fetch.py`. So there are at least 4 names for the same thing.

I feel like changing the name is going to need a `DeprecationWarning` because who knows who/what is using these various functions. So I think the best approach is probably to leave it as-is and make an issue of the problem.

I have added a trap in `fetch.py` to catch Falsey values though, that bit's ok I think. Tests are passing.

What do you think @berland?

Reply:
Yes, leaving the API as is can be good for now.